doc/devdocs/run-as-admin-detection.md

```diff
@@ -0,0 +1,51 @@
+
+# PowerToys and running as Administrator
+
+## Too long, Didn't Read 😁
+
+If you're running any application as an administrator and PowerToys is not, a few things may not work correctly when the elevated applications are in focus or trying to interact with a PowerToys feature like FancyZones.
```

Comment:
Maybe we can introduce the term "elevated" before using it, so we don't confuse the less technical users.

"If you're running any application as an administrator (aka elevated)"